priv/templates/boot.eex

```diff
@@ -2,6 +2,7 @@
 
 set -e
 
+command -v erl >/dev/null 2>&1 || { echo "erlang runtime not found. If erlang is installed check your $PATH. aborting.";  exit 1; }
```

Comment:
If that's the only place it's pulled from path does it make sense to just bail out if __erl is empty?